azext_connect/custom.py

```diff
@@ -22,14 +23,74 @@
 DEFAULT_CLI = get_default_cli()
 
 
-def connect(service, resource_group, para_list):
+# def connect(service, resource_group, para_list):
+#     if not resource_group:
+#         raise CLIError('--resource-group not specified')
+#     service_list = validate(service)
+#     check_resource(service_list, resource_group)
+#     para_dict = parseParameter(para_list)
+#     # print(para_dict)
+#     deploy(service_list, resource_group, para_dict)
+
+def connect(resource_group, aks = None, acr = None, webapp = None, sql = None):
     if not resource_group:
         raise CLIError('--resource-group not specified')
-    service_list = validate(service)
+    service_list = []
+    para_list = ''
+    connection_type = ''
+    url = ''
+    if aks:
+        service_list.append(SERVICE_MAP['aks'])
+        para_list = para_list + 'aks_name:' + aks + ' '
+        connection_type = connection_type + 'Service Principal'
+        url = url + 'https://aka.ms/AA861xl'
+    if acr:
+        service_list.append(SERVICE_MAP['acr'])
+        para_list = para_list + 'acr_name:' + acr + ' '
+    if webapp:
+        service_list.append(SERVICE_MAP['webapp'])
+        para_list = para_list + 'app_name:' + webapp + ' '
+    if sql:
+        service_list.append(SERVICE_MAP['sql'])
+        para_list = para_list + 'server:' + sql + ' '
+        print("Please input the database name.")
```

Comment:
This part can be put into a function so that we don't need to write all this stuff for other services.